guided-match/ddl.sql

```diff
@@ -0,0 +1,207 @@
+/*
+Title       Database creation script for FaT database design
+
+Date        May 11th 2020
+
+Author      Trevor Cummings
+
+Description This file is a script to create the tables for the Find a Thing database. The data model for FaT will be deployed in 
+            two separate databases. One will be a postgreSQL database for our relational tables and the other will be a Neo4j
+            for the graph database.
+            
+            I will comment out the tables which are for the graph database as the create table statements will be created
+            in a separate script
+
+
+*/
+
+/* Neo4J
+CREATE TABLE ccs_outcomes (
+   outcome_id                       INTEGER PRIMARY KEY,
+   journey_question_id              INTEGER NOT NULL,
+   outcome_evaluator_code           VARCHAR(10) NOT NULL, 
+   journey_answer_id                INTEGER NOT NULL,
+   ccs_support_ind                  CHAR(1) NOT NULL
+);
+*/
+
+
+/* Neo4J 
+CREATE TABLE domain_question_outcomes (
+   outcome_id                       INTEGER PRIMARY KEY,
+   journey_question_id              INTEGER NOT NULL,
+   outcome_evaluator_code           VARCHAR(10) NOT NULL, 
+   journey_answer_id                INTEGER NOT NULL,
+   modifier_journey_name            VARCHAR(10) NOT NULL
+);
+*/
+/* Neo4J
+CREATE TABLE explicit_question_outcomes (
+   outcome_id                       INTEGER PRIMARY KEY,
+   journey_question_id              INTEGER NOT NULL,
+   outcome_evaluator_code           VARCHAR(10) NOT NULL, 
+   journey_answer_id                INTEGER NOT NULL,
+   resulting_journey_question_id    INTEGER NOT NULL
+);
+*/
+
+/* Neo4J
+CREATE TABLE journey_answers ( --Need to check this one out with Dave H as attributes on model are different to descriptions on confluence page
+  journey_answer_id                INTEGER PRIMARY KEY,
+  journey_question_id              INTEGER NOT NULL,
+  answer_type                      VARCHAR(50),
+  answer_label                     VARCHAR(100),
+  default_ind                      CHAR(1), -- Y or N default value will be N as only one can be Y
+  valid_value                      VARCHAR(200),
+  url                              VARCHAR(200), -- notsure required anymore
+  regex_validation                 VARCHAR(2000)  -- notsure required anymore
+);
+*/
+
+CREATE TABLE journey_instance_answers (
+  journey_instance_answer_id        INTEGER PRIMARY KEY,
+  journey_instance_question_id      INTEGER NOT NULL,
+  journey_answer_id                 UUID,
+  answer_sequence                   SMALLINT,
+  answer_text                       VARCHAR(2000),
+  answer_date                       DATE,
+  answer_number                     NUMBER(18,4)

```

Comment:
`NUMERIC`

Reply:
Changed to numeric. Must have been on Oracle auto-pilot